x-pack/plugins/cloud_security_posture/public/common/hooks/use_subscription_status.ts

```diff
@@ -12,9 +12,10 @@ import { useKibana } from './use_kibana';
 
 const SUBSCRIPTION_QUERY_KEY = 'csp_subscription_query_key';
 
-export const useSubscriptionStatus = () => {
```

Comment:
nit: if we rename the only hook, should we also rename the file? not like we strictly follow one hool per file policy, so just a nit